config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-release-v1alpha4.yaml

```diff
@@ -295,6 +295,9 @@ presubmits:
         # docker-in-docker needs privileged mode
         securityContext:
           privileged: true
+        env:
+          - name: TEST_WINDOWS
```

Comment:
This is running against `- ^release-0.5$` which doesn't have containerd support.  I don't think the script would do anything with this but probably shouldn't include it